src/TBuffer.cpp

```diff
@@ -2417,7 +2439,7 @@ void TBuffer::translateToPlainText(std::string& incoming, const bool isFromServe
                         break;
                     case 2:
                         isTwoTCharsNeeded = true;
-                    // Fall-through
+                        [[clang::fallthrough]];
```

Comment:
Nice, but my clang is only 3.8 and doesn't support those C++17 features and although my GCC is 6.3 those aren't coming in until 7.x as I understand it...